js/app_common.js

```diff
@@ -21,6 +21,13 @@ if (settings.has("user.currency")) {
   settings.set("user.currency", "USD");
 }
 
+//default secondary currency
+if (settings.has("user.secondaryCurrency")) {
+  //do nothing because secondary currency already set
+} else {
+  settings.set("user.secondaryCurrency", "BRL");
```

Comment:
Secondary currency should be USD by default or maybe BTC since the primary is USD by default too.

Reply:
Indeed, BTC is a much better choice. Done.